subprojects/ide/ide.gradle

```diff
@@ -30,6 +30,7 @@ dependencies {
     compile libraries.inject
 
     testCompile libraries.xmlunit
+    testCompile libraries.equalsVerifier
```

Comment:
Ok, I had added this for myself to catch errors with the manually written equals/hashCode required by the DSL and decided to leave it in case the gradle team found it useful.

I've found EqualsVerifier quite useful on other projects as a low-effort way to catch issues with Java hashCode/equals (it catches things like possible NPEs, fields that were recently added without updating hashCode/equals, using a field in hashCode but forgetting it in equals, the semantics of getClass() vs instanceof in the presence of inheritance, and a few other useful things).  To check these edge cases in the integration tests might run over 1000 more lines of test code and take longer to execute.

If you think you will not use it often, I can remove it.  Please let me know what you decide.